### ISSUE TITLE
Underscore attributes change to dash attributes

### ISSUE THREAD

**Issue by sdayu** [qvantel/jsonapi-client#1]
I tried to set attributes of a model which contain underscore ('_'), but the `create` function change '_' to '.'. How to set the attributes with the underscore, I cannot find the example case.

```Python
>>> schema = {'user': {'properties': {'email': {'title': 'email', 'type': 'string'},
                                                          'first_name': {'title': 'first_name', 'type': 'string'},
                                                          'id': {'title': 'id', 'type': 'string'}, 
                                                          'last_name': {'title': 'last_name', 'type': 'string'}, 
                                                          'password': {'title': 'password', 'type': 'string'}, 
                                                          'status': {'default': 'unregister', 'title': 'status', 'type': 'string'},
                                                          'username': {'title': 'username', 'type': 'string'}},
                                      'required': ['email', 'first_name', 'last_name', 'password', 'username'], 
                                      'type': 'object'}}
>>> s = Session('http://localhost', schema=schema)
>>> a = s.create('user',
                            username='testusername',
                            email='testemail@email.com', 
                            password='testpassword',
                            first_name='test first name',
                            last_name='test_lastname')
There was extra data (not specified in schema): {'first-name': 'test first name', 'last-name': 'test_lastname'}
```
:

**Comment by tuomas2** [comment]
Hi @sdayu, could you please check if https://github.com/qvantel/jsonapi-client/pull/2 and test if it resolves your issue?

**Comment by tuomas2** [comment]
Fixed in #2

**Comment by sdayu** [comment]
Thank you for your responding,  it works as well.
